storage/client/blob/src/main/java/com/azure/storage/blob/package-info.java

```diff
@@ -3,7 +3,6 @@
 // Code generated by Microsoft (R) AutoRest Code Generator
 
 /**
- * This package contains the classes for StorageClient.
- * Storage Client.
+ * Package containing the classes for StorageClient.
```

Comment:
This was generated by codegen, this will be overwritten next time it is generated